internal/host/plugin/repository_host_set_test.go

```diff
@@ -60,8 +60,8 @@ func TestRepository_CreateSet(t *testing.T) {
 			name: "invalid-public-id-set",
 			in: &HostSet{
 				HostSet: &store.HostSet{
```

Comment:
Should we confirm that prefered endpoints column gets written to?

Reply:
Ahh yeah, thanks

Reply:
Added a test!